ResearchSuite/ResearchSuite/RSDTaskPath.swift

```diff
@@ -117,13 +116,12 @@ public final class RSDTaskPath : NSObject, NSCopying {
     /// a step.
     ///
     /// - note: The calling application is responsible for deleting this directory once the files
-    /// are processed by uploading them to a server or cloud service. Otherwise, since the the default
-    /// behavior is to save to application support, the app will slowly grow and take more memory.
+    /// are processed by uploading them to a server or cloud service. These files are **not**
```

Comment:
I think this should suggest that they delete the directory as soon as they've encrypted its contents rather than waiting until they've actually been uploaded.